ecpp/current.ecpp

```diff
@@ -362,7 +372,20 @@ std::vector<uint32_t> asset_ids;
 
             if( one_measurement.first == "status.ups" )
             {
-              siJson.addMember(one_measurement.first) <<= shared::upsstatus_to_string(uint16_t(one_measurement.second));
+                siJson.addMember(one_measurement.first) <<= shared::upsstatus_to_string(uint16_t(one_measurement.second));
+            }
+            // special treatment for battery status and battery charger status and battery alarm code (string values)
+            else if( one_measurement.first == "status.battery" || one_measurement.first == "status.battery.charger" ||
```

Comment:
instead of (re)listing metric exceptions here, simply code at line 372 (before if(status.ups)):
search .first (metricName) in str_measurements map... if found, addMember in siJson and continue....

Reply:
Take into account